tests/regression_tests/ncrystal/test.py

```diff
@@ -58,7 +59,6 @@ def pencil_beam_model(cfg, E0, N):
 
     return openmc.Model(geometry, materials, settings, tallies)
 
-
 class NCrystalTest(PyAPITestHarness):
```

Comment:
Per PEP8, top-level functions/classes should be preceded by two blank lines
```suggestion

class NCrystalTest(PyAPITestHarness):
```